dvc/remote/gdrive.py

```diff
@@ -324,10 +337,35 @@ def _create_remote_dir(self, parent_id, title):
 
     @gdrive_retry
     def _delete_remote_file(self, remote_id):
+        from pydrive2.files import ApiRequestError
+
         param = {"id": remote_id}
         # it does not create a file on the remote
         item = self.drive.CreateFile(param)
-        item.Delete()
+
+        try:
+            item.Trash() if self._trash_only else item.Delete()
+        except ApiRequestError as exc:
+            http_error_code = exc.error.get("code", 0)
+            if (
+                http_error_code == 403
+                and self.list_params["corpora"] == "drive"
+                and location(exc) == "file.permissions"
+            ):
+                raise DvcException(
+                    "Insufficient permissions to {}. You should have {} "
+                    "access level for the used shared drive. More details "
+                    "at {}".format(
```

Comment:
put a period at the end

Reply:
Sounds good; Initially, haven't put period because it looked ugly to me (please see the attached screenshot).
![2020-03-25_15-35-52-period](https://user-images.githubusercontent.com/1928902/77542210-813f5e00-6eae-11ea-934f-d5fb0cbacc79.png)